.github/workflows/ruby_on_rails.yml

```diff
@@ -7,8 +7,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rails_version: [5.0.0, 5.2.3, 6.0.0, master]
-        ruby_version: [2.4.x, 2.5.x, 2.6.x, 2.7.x]
```

Comment:
Why are we not testing older versions anymore?

Reply:
I was playing around with some CI issues on this branch. I'll end up looking into them elsewhere.